src/main/kotlin/matrix/glMatrix.kt

```diff
@@ -0,0 +1,58 @@
+package matrix
+
+import org.khronos.webgl.Float32Array
+import kotlin.js.Math
+
+open class glMatrix {
+    companion object {
+        /**
+         * Common utilities
+         * @module glMatrix
+         */
+        // Configuration Constants
+        var ARRAY_TYPE = Any()
```

Comment:
Is array type something copied from javascript? If yes, consider using generics.